README.md

```diff
@@ -272,6 +272,23 @@ kubectl logs pod/terminated-host-pod-name
 ```
 
 If you are permanently removing the host from the MarkLogic cluster, once the pod is terminated, follow standard MarkLogic administrative procedures using the administrative UI or APIs to remove the MarkLogic host from the cluster. Also, because Kubernetes keeps the Persistent Volume Claims and Persistent Volumes around until they are explicitly deleted, you must manually delete them using the Kubernetes APIs before attempting to scale the hosts in the StatefulSet back up again.
+
+## Deploying separate E/D node MarkLogic Cluster
```

Comment:
Suggested rewording:

## Deploying Multiple a MarkLogic Cluster with Multiple Groups

To deploy a MarkLogic cluster with multiple groups (separate E and D nodes for example) the `bootstrapHostName` and `group.name` must be configured in values.yaml or the values provided for these configurations using the --set flag while installing helm charts.
For example, if you want to create a MarkLogic cluster with three nodes in a "dnode" group and two nodes in an "enode" group, run the following helm commands:

```
helm install dnode-group ./charts/ --set group.name=dnode --set replicaCount=3
```
Once this deployment is complete, a MarkLogic cluster with three hosts in the should be running. 

To add the "enode" group and nodes to the cluster, the `bootstrapHostName` must be set to join the existing MarkLogic cluster. Any host in the other group can be used. For this example, set `bootstrapHostName` to `dnode-group-marklogic-0` with the following command:

```
helm install enode-group ./charts/ --set group.name=enode --set replicaCount=2 --set bootstrapHostName=dnode-group-marklogic-0
```
Once this deployment is complete, there will be a new "enode" group with two hosts in the MarkLogic cluster.

Reply:
> _Any host in the other group can be used_
As per the implementation, only first host is the other group(dnode) can be used as bootstrapHostName while joining the cluster.

Reply:
I think any hostname can be used. It doesn't have to be the first one. I know that the first one is the one marked as a "bootstrap" host in the MarkLogic cluster but that is for other things like coupling clusters. To join a cluster, you can do it against any host.

Reply:
I made a minor change to the wording and committed to the branch.